liferay-echo/liferay-portal
# LPD-33964 Reset custom styles to Clay Tree in Page Editor Browser

## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPD-33964)

### Changes

Reset custom styles to Clay Tree in Page Editor Browser

## Test scenario 

1. Create a content page
2. Go to the page tree
3. Check the changes

Before changes:
![image](https://github.com/user-attachments/assets/c6f9c4a2-2f28-4efe-81b4-318f63d31a02)

After changes:
![image](https://github.com/user-attachments/assets/d9d5ac06-04fc-4afb-9e43-6ff01b6d6447)
